### ISSUE TITLE
[FR] add commits statistics for merged PRs

### ISSUE THREAD

**Issue by bruceauyeung** [bruceauyeung/github-contrib-stats#3]
merged column should look like this:
```
       merged
--------------------
PRs    |   Commits
--------------------
x      |      y
--------------------
```

**Comment by bruceauyeung** [comment]
we use https://github.com/olekukonko/tablewriter to render table view. according to issue https://github.com/olekukonko/tablewriter/issues/1 , rowspan is not implemented yet.

**Comment by bruceauyeung** [comment]
implemented by commit 0489c6f5cd2a2ff52cb7100aec8167e54932bd33

now table view looks like this:
```
+--------------+------------+----------------+-------------+----------------+
|  USER NAME   | MERGED PRS | MERGED COMMITS | LGTM'ED PRS | NONLGTM'ED PRS |
+--------------+------------+----------------+-------------+----------------+
| bruceauyeung |         10 |             11 |           0 |              6 |
| tanshanshan  |          8 |              8 |           0 |              7 |
+--------------+------------+----------------+-------------+----------------+
```
